roles/pulp_webserver/vars/main.yml

```diff
@@ -1,4 +1,5 @@
 ---
-__pulp_apache_url: 'localhost'
-__pulp_webserver_apache_api_bind: "{{ pulp_api_bind.startswith('unix') | ternary(pulp_api_bind + '|http://' + __pulp_apache_url, 'http://' + pulp_api_bind ) }}"
-__pulp_webserver_apache_content_bind: "{{ pulp_content_bind.startswith('unix') | ternary(pulp_content_bind + '|http://' + __pulp_apache_url, 'http://' + pulp_content_bind ) }}"
+__pulp_webserver_apache_api_bind: "{{ pulp_api_bind.startswith('unix') | ternary(pulp_api_bind + '|http://' + __pulp_default_host, 'http://' + pulp_api_bind ) }}"
+__pulp_webserver_apache_content_bind: "{{ pulp_content_bind.startswith('unix') | ternary(pulp_content_bind + '|http://' + __pulp_default_host, 'http://' + pulp_content_bind ) }}"
+__pulp_webserver_nginx_api_bind: "{{ pulp_api_bind.startswith('unix') | ternary(pulp_api_bind + '|http://' + __pulp_default_host, 'http://' + pulp_api_bind ) }}"
+__pulp_webserver_nginx_content_bind: "{{ pulp_content_bind.startswith('unix') | ternary(pulp_content_bind + '|http://' + __pulp_default_host, 'http://' + pulp_content_bind ) }}"
```

Comment:
Should the variable `__pulp_default_host` not be called `__pulp_host`.
Calling it default sounds like it can be overwritten when using the role, which contradicts the `__` prefix.